pkgs/desktops/gnome-3/3.20/core/libpeas/default.nix

```diff
@@ -1,13 +1,15 @@
 { stdenv, fetchurl, pkgconfig, intltool, gnome3
-, glib, gtk3, gobjectIntrospection, python, pygobject3
+, glib, gtk3, gobjectIntrospection, python3, python3Packages, ncurses, python2, python2Packages
 }:
 
 stdenv.mkDerivation rec {
   inherit (import ./src.nix fetchurl) name src;
 
+  configureFlags = [ "--enable-python3" "--enable-python2" ];
+
   buildInputs =  [
-   intltool pkgconfig glib gtk3 gobjectIntrospection python pygobject3
-   gnome3.defaultIconTheme
+   intltool pkgconfig glib gtk3 gobjectIntrospection python3 python3Packages.pygobject3 python2
+   gnome3.defaultIconTheme ncurses
```

Comment:
I don't know what this derivation exactly does, but if it needs to use Python during runtime, then the environment (interpreter + Python packages) need to be `propagatedBuildInputs`.